src/app.ts

```diff
@@ -157,6 +157,13 @@ async function downloadPage(title: string, link: string, doToggleMenu: boolean):
 
   await page.addStyleTag({ content: 'div[class^="styles__PrevNextButtonWidgetStyled"], div[class^="styles__Footer"], nav { display: none !important; }' });
 
+  await page.evaluate(() => {
+    const elements = document.getElementsByClassName('Button-ktk5iw-0 CircleButton-f4i43u-0 styles__AnimationPlus-sc-2az9w3-13 dqWxjg')
```

Comment:
I believe this class name is dynamic. It might not be same for everyone. So, it would be safe to handle using node.

Reply:
@shihabmridha just commented same. Can you please look into it. I am not so sure about it.